pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/analytics/PlaybackSessionManager.kt

```diff
@@ -45,50 +47,69 @@ class PlaybackSessionManager {
         val sessionId = UUID.randomUUID().toString()
     }
 
+    /**
+     * Session info
+     *
+     * @property session The [Session]
+     * @property position The position in milliseconds when a session changes occurs.
+     */
+    data class SessionInfo(
+        val session: Session,
+        val position: Long,
+    )
+
     /**
      * Listener
      */
     interface Listener {
         /**
          * On session created
          *
-         * @param session
+         * @param session The newly created [Session].
          */
         fun onSessionCreated(session: Session) = Unit
 
         /**
-         * On current session
+         * On session finished The session won't be current any more.
          *
-         * @param session
+         * @param session The destroyed [Session].
          */
-        fun onCurrentSession(session: Session) = Unit
+        fun onSessionDestroyed(session: Session) = Unit
 
         /**
-         * On session finished
+         * On current session changed from [oldSession] to [newSession].
+         * [onSessionCreated] with [oldSession] is called right after.
```

Comment:
I guess this should either be:
```suggestion
         * [onSessionCreated] with [newSession] is called right after.
```
Or:
```suggestion
         * [onSessionDestroyed] with [oldSession] is called right after.
```

Based on the following, I'd say it's the second option:
https://github.com/SRGSSR/pillarbox-android/blob/037a904047d8509caa4bab5920acbcbb4f4984b3/pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/analytics/PlaybackSessionManager.kt#L103-L110